packages/deploy-utils/src/prepare-prod-deploy/prepare-prod-deploy.test.ts

```diff
@@ -93,9 +93,11 @@ my commit messages`,
     });
 
     it("redeploys tag when user confirms prompt", async () => {
-      vi.mocked(prompts).mockResolvedValueOnce({
-        value: true,
-      });
+      vi.mocked(prompts)
+        .mockResolvedValueOnce({ value: false })
```

Comment:
nit: would be helpful if there was a comment describing which prompt the resolved value refers to.